python/keepsake/project.py

```diff
@@ -13,7 +13,7 @@
 
 
 MAX_SEARCH_DEPTH = 100
-DEPRECATED_REPOSITORY_DIR = ".replicate/storage"
+DEPRECATED_REPOSITORY_DIR = ".keepsake/storage"
```

Comment:
```suggestion
DEPRECATED_REPOSITORY_DIR = ".replicate/storage"
```

Is this even used any longer?

Reply:
nope it's not, deleting.